ansible/roles/dev/tasks/main.yml

```diff
@@ -175,3 +175,23 @@
 - name: Install pulp-dev library (system-wide)
   command: pip3 install -e {{pulp_devel_dir}}/devel/pulp-dev
   when: "{{ ['devel'] | issubset(pulp_available_repositories) }}"
+
+- name: install systemd files
+  copy:
```

Comment:
Ok, after looking at it more I think what you need to do is put

```dest: "/usr/systemd/system/{{ item }}```

I believe the file module uses different semantics than the copy module so you've got to actually specify the name of the destination file instead of the module making the assumption that it remains the same.